bootstrap_salt/fab_tasks.py

```diff
@@ -32,6 +32,21 @@ def aws(x):
 
 
 @task
+def environment(x):
+    env.environment = str(x).lower()
+
+
+@task
+def application(x):
+    env.application = str(x).lower()
+
+
+@task
+def config(x):
+    env.config = str(x).lower()
```

Comment:
Or another option. Put this in the top your fabfile:

```python
from fabric.api import env, task

env.application = "my-app"

@task
def dev():
    """
    Select the dev stack
    """
    env.config = "foo/dev.yml"
    env.environment = "dev"

# …
```

These parameters don't really need to be specified on the command line each time as they are largely static.